src/src/com/microsoft/aad/adal/HttpWebRequest.java

```diff
@@ -182,8 +182,10 @@ public HttpWebResponse send() {
                 Logger.v(TAG, "Response is received");
                 response.setBody(responseBody);
                 response.setResponseHeaders(mConnection.getHeaderFields());
-            } catch (Exception e) {
-                Logger.e(TAG, "Exception:" + e.getMessage(), " Method:" + mRequestMethod,
+            } catch (InterruptedException ignore) {
```

Comment:
While we are here, let's change the name of this variable from 'ignore' to 'e'.  There's not reason for it to be different from the other exception variable names and the difference seems to imply there is funny going on in this catch block.